### DISCUSSION TITLE
timer for remote control

### DISCUSSION THREAD

**Discussion by garfield50** [amidaware/tacticalrmm#438] [Ideas]
hi all,

Your work is amazing !!! continue like that !!!! just an idea, can you use a timer for the remote control in the audit log to know the duration of the remote control.

Sorry, my English is bad because I am French.

Thank you.

**Comment by dinger1986** [comment]
this is recorded in mesh at the moment, does it need recorded in tactical as well?

**Reply by garfield50** [reply]
yes, because i can give a log with connection duration every months to my customer. to provide proof of connection

**Reply by dinger1986** [reply]
ok, you would want to open a feature request, I guess it would really be under feature requests but I would make it as a reports feature request

**Reply by garfield50** [reply]
sorry if I entered the wrong section!
